File: 2.0/js/campaigns/postCallCampaignSound.js
'strict'

const CallfireClient = require('callfire-api-client-js');
const client = new CallfireClient('api-login', 'api-password');

client.ready(() => {
    client.campaigns.postCallCampaignSound({
      body: {
        name: 'Sound 1',
        toNumber: '12135551122'
      }
    })
      .then((response) => {
        console.log(response.obj);
      })
      .catch((err) => {
        console.log('request error ' + err.data);
      });
  },
  (clientError) => {
    console.log('client error ' + clientError);
  }
);

File: 2.0/python/campaigns/postCallCampaignSound.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.campaigns.postCallCampaignSound(
    body={
        'name': 'Sound 1',
        'toNumber': '12135551122'
    }
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)
